src/desktop/apps/article/components/__tests__/InfiniteScrollNewsArticle.test.js

```diff
@@ -14,19 +14,22 @@ describe('<InfiniteScrollNewsArticle />', () => {
   let props
   let article
   let nextArticle
+  let clock
 
-  before((done) => {
+  before(done => {
     benv.setup(() => {
       benv.expose({ $: benv.require('jquery'), jQuery: benv.require('jquery') })
       sd.APP_URL = 'http://artsy.net'
       sd.CURRENT_PATH =
         '/news/artsy-editorial-surprising-reason-men-women-selfies-differently'
       sd.CURRENT_USER = { id: '123' }
+      clock = sinon.useFakeTimers()
       done()
```

Comment:
Ooh very interesting -- I wonder if we can override dates with jest too, there are a few problematic Reaction tests that could use this.